vermouth/data/force_fields/elnedyn22p/aminoacids.ff

```diff
@@ -687,3 +687,8 @@ SC1 >SC1 1 0.24 {"comment": "Disulfide bridge"}
 ;[ features ]
 ;disulfide
 
+; Exclusions between the charged terminii and the charge dummies
+[ link ]
+resname $protein_resnames
+[ exclusions ]
+BB {"atype": "Qd|Qa"} XX {"atomname": "SCP|SCN"}
```

Comment:
@pckroon uhhh this works? I didn't know you could make the atomname attribute a variable as will. You know that this changes the whole game of making links?

Reply:
Apparently it works. There's no magic code to make this possible even, so yes, it's general. Not sure how you would deal with the case where *any* atomname is acceptable though.